proxy/fadein.go

```diff
@@ -0,0 +1,53 @@
+package proxy
+
+import (
+	"math"
+	"math/rand"
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/routing"
+)
+
+func returnFast(rt *routing.Route) bool {
```

Comment:
The name sounds a bit weird.
Maybe: `noFadeIn` or `shouldFadeIn`

or drop the func and change the logic below to:
```
if rt.BackendType != eskip.LBBackend || rt.LBFadeInDuration <= 0 { return endpoints }
```

Reply:
Do we need to check backend type at all? This code is only called for loadbalanced backends anyway as it passes endpoints